core/templates/pages/exploration-editor-page/feedback-tab/services/thread-data-backend-api.service.ts

```diff
@@ -254,7 +254,8 @@ export class ThreadDataBackendApiService {
     }).toPromise().then((response: ThreadMessages) => {
       if (updatedStatus) {
         this.openThreadsCount += (
-          (newStatus === ExplorationEditorPageConstants.STATUS_OPEN) ? 1 : -1);
+          (newStatus === ExplorationEditorPageConstants.STATUS_OPEN) ? 1 : (
+            oldStatus === ExplorationEditorPageConstants.STATUS_OPEN) ? -1 : 0);
```

Comment:
I would expand this into a proper ifs (or at least one) because nested onliners are quite hard to read.
```typescript
if (newStatus === ExplorationEditorPageConstants.STATUS_OPEN) {
  this.openThreadsCount += 1;
} else {
  this.openThreadsCount += (
    oldStatus === ExplorationEditorPageConstants.STATUS_OPEN ? -1 : 0)
}
```

Reply:
Done!